index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Austin Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header>
+            <div class="logo-container">
+                <a href="#"><h3 id="logo">My Logo</h3></a>
+            </div>
+            <nav>
+                <label class="hamberger-btn" >
+                <img class="hamberger" id="ham" src="img/IconMenu.png">
+                </label>
+            </nav>
+        </header>
+        <section class="banner-section" id="Home">
+            <div class="container">
+                <h1 id="myName">
+                    Hey There.
+                    <br>
+                    I'm Austine.
+                </h1>
+                 <p id="title">I'm a Software Developer</p>
+                 <p id="description">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                   don’t hesitate to contact me.</p>
+                <div class="socials">
+                   <ul class="iconsSection">
+                        <li class="icons">
+                            <a href="https://github.com/Obimbo07" ><img src="img/github.svg" alt="GitHub" /></a>
+                        </li>
+                        <li class="icons">
+                            <a href="https://www.linkedin.com/in/austin-obimbo-9a613623a/"  ><img src="img/Linkedin icon.svg" alt="LinkedIn" /></a>
+                        </li>
+                        <li class="icons">
+                            <a href=""><img src="img/angel list.svg" alt="angel" /></a>
+                        </li>
+                        <li class="icons">
+                            <a href=""><img src="img/twitter.svg" alt="twitter" /></a>
+                        </li>
+                        <li class="icons">
+                            <a href=""><img src="img/medium.svg" alt="Medium" /></a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] As required for the project to stick as much as possible to the original design, I think it would be better you improve your current design to looks similar to your chosen Figma design (e.g.: `Fonts`(family, size, weight,...), padding, margin, alignment, spacing between and around elements,...). Make sure you check all details on the template and match the expected design.

| Figma design | Your Design |
| ------------------ | ---------------- |
| ![Screenshot from 2023-07-05 21-57-17](https://github.com/Obimbo07/Portfolio-2/assets/83042742/ca360f18-fbb0-40af-b93c-e46ac3524f45) | ![Screenshot from 2023-07-05 21-58-19](https://github.com/Obimbo07/Portfolio-2/assets/83042742/d78f7a06-80f2-4df9-9598-533e9e7beea9) |